packages/tasit-identity-contract/src/GnosisSafeUtils.js

```diff
@@ -0,0 +1,149 @@
+import Action from "tasit-action";
+const { Contract } = Action;
+import { ethers } from "ethers";
+
+const { utils: ethersUtils } = ethers;
+const { bigNumberify } = ethersUtils;
+
+// 100 gwei
+const GAS_PRICE = bigNumberify(`${1e11}`);
+
+export default class GnosisSafeUtils {
+  #contract;
+
+  constructor(contract) {
+    this.#contract = contract;
+  }
+
+  // Sign a hash using N signers
+  // Based on: https://github.com/gnosis/safe-contracts/blob/102e632d051650b7c4b0a822123f449beaf95aed/test/utils.js#L93
+  multiSign = (wallets, hash) => {
```

Comment:
We should add a test for this util

Reply:
OK.

Reply:
Can come in a future PR, though

Reply:
Extracted to: https://github.com/tasitlabs/TasitSDK/issues/237

Reply:
Thanks. Resolving